src/webgl/context.js

```diff
@@ -196,17 +235,18 @@ export function glContextWithState(gl, {scissorTest, framebuffer}, func) {
  * @return {Object} - 'vendor' and 'renderer' string fields.
  */
 export function glGetDebugInfo(gl) {
-  const info = gl.getExtension('WEBGL_debug_renderer_info');
-  // We can't determine if 'WEBGL_debug_renderer_info' is supported by
-  // checking whether info is null here. Firefox doesn't follow the
-  // specs by returning null for unsupported extension. Instead,
-  // it returns an object without GL_UNMASKED_VENDOR_WEBGL and GL_UNMASKED_RENDERER_WEBGL.
-  return {
-    vendor: (info && info.UNMASKED_VENDOR_WEBGL) ?
-      gl.getParameter(info.UNMASKED_VENDOR_WEBGL) : 'unknown',
-    renderer: (info && info.UNMASKED_RENDERER_WEBGL) ?
-      gl.getParameter(info.UNMASKED_RENDERER_WEBGL) : 'unknown'
-  };
+  return getGLContextInfo(gl);
```

Comment:
Why do we create a new method getGLContextInfo() here and make glGetDebugInfo() simply call it?